<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>79</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditRegexImageFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>.+.bmp</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetImageFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>491</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>901</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditOutFolder">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>751</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditImageFolder">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>751</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenImageFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Image Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenOutFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Out Folder</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameMode">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>161</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxShowInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show input</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show output</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxScaleBase">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelScaleBase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale base</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxScalePower">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelScalePower">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale Power</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowTiffInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Tiff Info</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMatInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Mat Info</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAutocleanOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto clean output text</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLoadAnydepth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Anydepth</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSaveOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Output</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDisplayRange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFixMinDisp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>65535.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>64.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFixMaxDisp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>65535.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>64.000000000000000</double>
     </property>
     <property name="value">
      <double>1024.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowHist">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Histogram</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxHistScale_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistScaleHeight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistScaleCoef">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string>*10^</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelDirHistBarWidth_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Bar Width</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistBarWidth">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>380</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxImRotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Rotation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Mask</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameMode_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxLargeBlackPatches">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Czarna plama duża</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSmallBlackPatches">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Czarna plama mała</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxBlackEnd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Czarna końcówka</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxRotEnd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nadgnita końcówka</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxWhiteFungus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Białe ognisko grzybni</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxRotOutEnd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nadnicie paza końcówką</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGreenHead">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Zielona główka</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCrackTravsverse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pęknięcie poprzeczne</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCrackLongitudial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pęknięcie podłóżne</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCurvature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Krzywizna</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSplit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rozdwojenie</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOtherShapeDeformation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Inna deformacja kształtu</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxBiteInsect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nadgryzienie od owadów</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxBiteAnimal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nadgryzienie od gryzoni</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSoilure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Zanieczyszczenie ziemią</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Problem</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Problem</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Problem</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>511</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBoxOutputMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTransparency">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAutoRotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto Rotate</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxCropY">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4000</number>
     </property>
     <property name="value">
      <number>1400</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCreateMazdaFiles">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CreateMazdaFiles</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSelectMazdaFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Select MaZda Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSelectMazdaOptions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Mazda options</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMaZdaFolder">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOptionsFile">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDefectName">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxDefectNr">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>14</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCreateImages">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Create images</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
